Library/Homebrew/cmd/info.rb

```diff
@@ -303,6 +305,17 @@ def info_formula(formula)
         ]
         if kegs.empty?
           puts "Not installed"
+          if args.github_manifest? && !(bottle = formula.bottle).nil?
+            begin
+              bottle.fetch_tab(quiet: !args.debug?)
+              bottle_size = bottle.bottle_size
+              puts "Bottle size: #{disk_usage_readable(bottle_size)}" if bottle_size.positive?
+              installed_size = bottle.installed_size
+              puts "Installed size: #{disk_usage_readable(installed_size)}" if installed_size.positive?
```

Comment:
Currently looks like
```
Bottle size: 5MB
Installed size: 16.0MB
```

Apt uses `Download-Size` and `Installed-Size`.

https://archlinux.org/packages shows `Package Size` and `Installed Size`.